beacon-chain/sync/validate_blob.go

```diff
@@ -92,13 +93,48 @@ func (s *Service) validateBlob(ctx context.Context, pid peer.ID, msg *pubsub.Mes
 	switch parentStatus := s.handleBlobParentStatus(ctx, parentRoot); parentStatus {
 	case pubsub.ValidationIgnore:
 		log.WithFields(blobFields(blob)).Debug("Ignored blob: parent block not found")
```

Comment:
This log should change to something like `Parent block not found - saving blob to cache`